packages/apidom-ns-workflows-1/src/refractor/visitors/workflows-1/StepOnFailureVisitor.ts

```diff
@@ -0,0 +1,28 @@
+import stampit from 'stampit';
```

Comment:
should be located in `workflows-1/step/OnFailureVisitor.ts` and the name of visitor should be `OnFailureVisitor`.

Reply:
Thanks - nice spot. Location changed